xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/layouts/table/LayoutTable.vue

```diff
@@ -117,6 +117,16 @@ export default {
     canAddEntry () { return this.logic.canAddEntry(); },
   },
 
+  beforeMount () {
+    this.logic.loadTranslations({
+      componentName: "layout-table",
+      prefix: "livedata.layout.table",
+      keys: [
+        "title",
+      ],
+    });
+  },
```

Comment:
Not needed because the layout title is already translated (it is part of the live data descriptor).